ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -183,6 +183,7 @@ wan_rr:
           wan_carrier: Another-ISP
           wan_circuit_id: 999
           ip_address: 10.9.9.9/31
+      ssl_profile: profileA
```

Comment:
this is too unclear what this is relative too - it is for stun so maybe `stun_ssl_profile`?